lib/start-controller.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const BaseController = require('./base-controller');
+
+module.exports = class StartController extends BaseController {
+
+  constructor(options) {
+    super(options);
+  }
+
+  getValues(req) {
+    req.sessionModel.reset();
+    super.successHandler.apply(this, arguments);
```

Comment:
when using super.methodName, you don't need to pass the scope. I know this way you can just pass `arguments` but I think I prefer the explicit `super.successHandler(req, res, callback)` with the extra args in the class method, thoughts?